doc/sphinx/addendum/nsatz.rst

```diff
@@ -32,6 +32,36 @@ Nsatz: tactics for proving equalities in integral domains
 
    doing automatic introductions.
 
+   `radicalmax`
+     bound when searching for r such that
+     :math:`c (P−Q) r = \sum_{i=1..s} S_i (P i − Q i)`
+
+   `strategy`
+     gives the order on variables :math:`X_1,\ldots,X_n` and the strategy
+     used in Buchberger algorithm (see :cite:`sugar` for details):
+
+       * strategy = 0: reverse lexicographic order and newest s-polynomial.
+       * strategy = 1: reverse lexicographic order and sugar strategy.
+       * strategy = 2: pure lexicographic order and newest s-polynomial.
+       * strategy = 3: pure lexicographic order and sugar strategy.
```

Comment:
```suggestion
       * `strategy := 0%Z`: reverse lexicographic order and newest s-polynomial.
       * `strategy := 1%Z`: reverse lexicographic order and sugar strategy.
       * `strategy := 2%Z`: pure lexicographic order and newest s-polynomial.
       * `strategy := 3%Z`: pure lexicographic order and sugar strategy.
```

Reply:
Got it.  Kind of weird you have to put "%Z" after something that's not even used as a number.

Reply:
Yes. We are just documenting this but honestly, I'm seriously scared by the way this tactic is taking its arguments.

Reply:
Something more high-level could indeed be done. I don't know how much these arguments are used though in practise and whether a change could be done without too much worries. Cc @thery